src/ctia/stores/es/mapping.clj

```diff
@@ -256,6 +256,12 @@
     :columns {:enabled false}
     :rows {:enabled false}}})
 
+(def note-related-entity
+  {:type "object"
+   :properties
+   {:entity_type token
+    :entity_id token}})
```

Comment:
We tend to put entity specific mappings in the corresponding ns.

Reply:
done